gradoop-flink/src/main/java/org/gradoop/flink/algorithms/fsm/transactional/gspan/GSpanBase.java

```diff
@@ -107,6 +108,6 @@ public GSpanBase(FSMConfig fsmConfig) {
         .groupBy(0)
```

Comment:
Suggestion. By only performing a text search within the review, I didn't manage to find a part of the code where the actual gSpan is used (most probably, that's my problem). If this code is actually used, I'll tell you where and why.